rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    book.rentals << self
```

Comment:
- [ ] Kindly convert the `rentals` property in the `Book` class to `attr_accessor` instead of `attr_reader` so you can add value to it inside the `initialize` in the `Rental` class.